src/main/java/innohack/gem/service/extract/ExcelExtractor.java

```diff
@@ -1,13 +1,99 @@
 package innohack.gem.service.extract;
 
+import com.beust.jcommander.internal.Lists;
 import innohack.gem.entity.GEMFile;
 import innohack.gem.entity.extractor.ExtractConfig;
 import innohack.gem.entity.extractor.ExtractedRecords;
+import innohack.gem.entity.extractor.TimestampColumn;
+import innohack.gem.entity.feature.AbstractFeature;
+import innohack.gem.entity.feature.ExcelFeature;
+import innohack.gem.entity.rule.ParamType;
+import innohack.gem.entity.rule.Parameter;
+import java.text.ParseException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class ExcelExtractor extends AbstractExtractor {
 
+  public ExcelExtractor() {
+    this(null, null);
+  }
+
+  public ExcelExtractor(String sheetName, String columnNames) {
+    Parameter param1 = new Parameter("Sheet Name", "", ParamType.STRING_LIST, sheetName);
```

Comment:
Sheet name should be string  instead of string_list type Assuming we are extracting from one sheet only per group

Reply:
ok done.